src/Performance/PerformanceJSON.ts

```diff
@@ -51,6 +86,31 @@ namespace PerformanceJSON {
         };
     }
 
+    interface MPTStatisticsBreakdownItem {
+        AsOfDate: string;
+        TrailingTimePeriod: TrailingTimePeriod;
+        DataFrequency: string;
+        Portfolio: MPTStatisticsPortfolio;
+    }
+
+    interface MPTStatisticsPortfolio {
```

Comment:
Missing all `Gross*` props when `IncludeGrossNetReturns` is defined (the same problem as with RiskStats converter):

<img width="652" height="704" alt="Image" src="https://github.com/user-attachments/assets/89f261b9-713d-4bc0-8e2c-1c0238570e08" />